project-proposals/stage-2/stage-2-questionnaire.md

```diff
@@ -0,0 +1,60 @@
+# Stage 2: Incubation
+
+*As a project intending to move to stage 2 of the OpenRail incubation process, please fill out this questionnaire and send it to the Technical Committee. See the [incubation process](../../incubation-process.md) for details about criteria and requirements for new projects and how it works to get new projects into the association.*
+
+*Copy this template and fill in your answers to the questions in the sections below.*
+
+## Health metrics
+
+*How to assess project health depends on the context of the concrete project. Please answer the questions below and add any additional explanations or data you see as useful to show that the project has a healthy community and development activity.*
+
+### How many committers does the project have?
+
+
+### How many different organizations are contributing to the project?
```

Comment:
How the project can retrieve this information?

Reply:
One way would be to look at the email addresses of committers. One example of this kind of statistics is the [development statistics for the Linux kernel](https://lwn.net/Articles/972605/).